### PR TITLE
Fucus and Vimeo icons | New

### PR DESCRIPTION
**Pull request by ZilvinasMaciulis**

### CODE REVIEW

**Review by TalZa1810 — APPROVED** [review]

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-icons-common.pr_2026/

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 1]
```diff
@@ -1965,6 +1972,12 @@ const generalIconsMetadata: Array<IconMetadata> = [
     tags: ["social"],
     sizes: { "18": "WhatsappSmall", "24": "Whatsapp" },
   },
+  {
+    title: "Vimeo",
+    category: "Brands",
+    description: "Button - redirects to Vimeo social network",tags: ["social", "video"],
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/general/metadata.ts:1978]: you've forgotten to add tags here.

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 2]
```diff
@@ -1965,6 +1972,12 @@ const generalIconsMetadata: Array<IconMetadata> = [
...
+    sizes: { "18": "VimeoSmall", "24": "ViemeoSmall" },
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/general/metadata.ts:1979]:
```suggestion
    tags: [ "share", "video", "upload"],
    sizes: { "18": "VimeoSmall", "24": "ViemeoSmall" },
```

File: packages/wix-ui-icons-common/src/general/metadata.ts [hunk 3]
```diff
@@ -1465,6 +1465,13 @@ const generalIconsMetadata: Array<IconMetadata> = [
     tags: ["flip", "reflect"],
     sizes: { "18": "MirrorVerticalSmall", "24": "MirrorVertical" },
   },
+  {
+    title: "Focus",
+    category: "Composer Actions",
+    description: "Button - automatically crops image to region of interest",
+    tags: ["crop", "image", "focal point"],
```
Comment (TalZa1810) [inline packages/wix-ui-icons-common/src/general/metadata.ts:1472]:
```suggestion
    tags: ["crop", "image", "region", "focal point"],
```

**Pull request merged**